doc/reference/testing.rst

```diff
@@ -111,6 +111,90 @@ supported.
 
 .. _unittest documentation: https://docs.python.org/2/library/unittest.html
 
+Test tags
+---------
+
+In Odoo, Python tests can be tagged to facilitate the test selection when running tests.
+
+By default, tests are tagged 'standard', 'at_install' and also with a special tag that is the technical name of the module where the test lies.
+
+A command line switch argument exists to select the tests to start. This argument is named ``--test-tags`` and defaults to ``+standard``. It means that when Odoo is started with the ``--test-enable`` switch, all the standard tests will be executed even if the tests were not explicitly tagged.
```

Comment:
``:option:`--test-tags <odoo-bin --test-tags>` `` (and same for the other commands following)